src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -57,7 +58,13 @@ const OptionRowLHN = (props) => {
     const optionItem = SidebarUtils.getOptionData(props.reportID);
     const isPinned = _.get(optionItem, 'isPinned', false);
 
+    const isScreenFocused = useIsFocused();
     React.useEffect(() => {
+        // Don't perform this side-effect if the SidebarScreen is not focused
+        if (!isScreenFocused) {
+            return;
+        }
+
```

Comment:
I commented [here](https://github.com/Expensify/App/issues/20422#issuecomment-1581918365) for alternative solution.

Reply:
Thanks, I actually was able to remove this effect entirely – seems it wasn't necessary, or isn't anymore